src/IndividualReview.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { useLocation, useParams } from "react-router";
+import { Link } from "react-router-dom";
+
+const IndividualReview = () => {
+  let { reviewId } = useParams();
+  const [review, setReview] = useState("");
```

Comment:
Best to keep the default state value as the same data type (e.g. an empty object)